rocketpy/simulation/monte_carlo.py

```diff
@@ -143,7 +152,8 @@ def __init__(
         self._last_print_len = 0  # used to print on the same line
 
         self.export_list = self.__check_export_list(export_list)
-        self.export_function = export_function
+        self._check_data_collector(data_collector)
```

Comment:
Optional fix, but to keep consistency with the `__check_export_list`, I would double underscore the method to `__check_data_collector`.

Reply:
I implemented with double underscores, but could not call them on the test suite.